drivers/atwinc15x0/atwinc15x0_netdev.c

```diff
@@ -144,6 +308,137 @@ typedef union {
 
 static bool _rssi_info_ready = false;
 
+static void _atwinc15x0_handle_M2M_WIFI_RESP_SCAN_DONE(tstrM2mScanDone* scan_done)
+{
+    DEBUG("%s scan done, %d APs found\n", __func__, scan_done->u8NumofCh);
+    if (scan_done->u8NumofCh > 0) {
+        /* read the first scan result record */
+        m2m_wifi_req_scan_result(0);
+    }
+    else {
+        /* no results */
+        _atwinc15x0_set_scanning(atwinc15x0, false);
+#if IS_USED(MODULE_ATWINC15X0_DYNAMIC_SCAN)
+        wifi_scan_list_t result = WIFI_SCAN_LIST_INITIALIZER(NULL);
+        _wifi_scan_result_callback(&result);
+#endif
```

Comment:
Is it possible to use the following here?
```suggestion
        if (IS_USED(MODULE_ATWINC15X0_DYNAMIC_SCAN)) {
            wifi_scan_list_t result = WIFI_SCAN_LIST_INITIALIZER(NULL);
            _wifi_scan_result_callback(&result);
        }
```